static/js/app.ts

```diff
@@ -225,6 +223,22 @@ function clearErrors(editor: AceAjax.Editor) {
   }
 }
 
+export function stopit() {
+  // We bucket-fix stop the current program by setting the run limit to 1ms
```

Comment:
Does this "bucket fix" need an improvement in the future (then maybe add an issue), or are we ultimately happy with it? (in that case, can we remove the comment?)